js/app.js

```diff
@@ -76,6 +81,64 @@
 				url: '/video/:id',
 				parent: 'root',
 				templateUrl: 'pages/video-detail/video-detail.view.html',
+			})
+			.state('videoDetailEdit', {
+				url: '/video/:id/edit',
+				parent: 'root',
+				templateUrl: 'pages/video-detail-edit/video-detail-edit.view.html',
 			});
 	}
+
+	function setupMaterialTheming($mdThemingProvider) {
```

Comment:
consider moving this function to another file

Reply:
We could even make it part of flavour files 🤖